### PR TITLE
changes to make instrumentation work with mutli tenant configuration

### PR DESCRIPTION
**Pull request by offerakrabi**
moved performance data under 'internal'

### CODE REVIEW

File: lib/handler.js [hunk 1]
```diff
@@ -127,7 +127,7 @@ Handler.prototype.handleRequest = function (request, callback) {
         callback(err, response.response);
     };
     response = new Response(responseCallback);
-    response.response.performance = request.performance;
+    response.response.performance = request.internal.performance;
```
Comment (erezbi) [inline lib/handler.js]: should we add 'internal' and have:
response.response.internal.performance
instead of:
response.response.performance
?

**Comment by erezbi** [comment]
@offerakrabi was this tested?

**Comment by offerakrabi** [comment]
@erezbi 
only manually by me.

**Comment by erezbi** [comment]
This is not good enough. This will be in production, and will be turned on and off in production.
We need this verified properly.

**Pull request merged**